### ISSUE TITLE
Installation script does not work on Raspberry Pi

### ISSUE THREAD

**Issue by screendriver** [Schniz/fnm#302]
I figured out that with #222 and #276 `fnm` should work on a Raspberry Pi but unfortunately it doesn't. On a fresh system I tried:

```sh
pi on raspberrypi in ~ 
✖ curl -fsSL https://fnm.vercel.app/install | bash
Checking dependencies for the installation script...
Checking availability of curl... OK!
Checking availability of unzip... OK!
Downloading https://github.com/Schniz/fnm/releases/latest/download/fnm-linux.zip...
######################################################################## 100.0%##O#- #                ######################################################################## 100.0%##=O#- #               ######################################################################## 100.0%
Installing for Zsh. Appending the following to /home/pi/.zshrc:

  # fnm
  export PATH=/home/pi/.fnm:$PATH
  eval "`fnm env`"

In order to apply the changes, open a new terminal or run the following command:

  source /home/pi/.zshrc
```

But then I get a

```sh
pi on raspberrypi in ~
❯ fnm --version
zsh: exec format error: fnm
```

**Comment by Schniz** [comment]
There are prebuilt binaries for ARM, yet the installation script does not support ARM right now. I don't really have an ARM machine so I haven't tested it at all.

I'm going to open a PR with a custom script for you to test. give me 5 min 😁

**Comment by screendriver** [comment]
@Schniz now it works 🆒 But now I have the problem

```sh
❯ fnm install 15 
Installing Node v15.0.1

pi on raspberrypi in ~ took 5s 
❯ which node
node not found

pi on raspberrypi in ~ 
❯ node --version
zsh: command not found: node
```

**Comment by Schniz** [comment]
what does `fnm exec --using=15 -- node --version` prints?

**Comment by screendriver** [comment]
```sh
✖ fnm exec --using=15 -- node --version
v15.0.1
```

**Comment by Schniz** [comment]
Try: `fnm use 15` and then `node -v`

**Comment by screendriver** [comment]
> Try: `fnm use 15` and then `node -v`

🤦 

Sorry, my fault. Now it's working. Awesome! I guess I thought that on a fresh installation a `fnm install 15` also sets this node version as default like `nvm` is doing it I believe.

Ah and by the way: `fnm install v15` does not work. The same for `.nvmrc`. It only works when I drop the `v`. Maybe you are already aware of it 🙂

**Comment by Schniz** [comment]
> Ah and by the way: `fnm install v15` does not work. The same for `.nvmrc`. It only works when I drop the `v`. Maybe you are already aware of it 🙂

Yeah, fixed in #301 😁 will release soon.

> I guess I thought that on a fresh installation a `fnm install 15` also sets this node version as default like `nvm` is doing it I believe.

You're right. Working on it!

**Comment by screendriver** [comment]
🆒 👍
